website/docusaurus.config.js

```diff
@@ -41,7 +41,25 @@ module.exports = {
         { to: 'https://github.com/h2oai/wave/discussions', label: 'Discuss', position: 'left' },
         { to: 'https://h2oai.github.io/h2o-ai-cloud/', label: 'Enterprise', position: 'left', },
         { to: 'https://github.com/h2oai/wave/releases', label: 'Download', position: 'right', },
-        { href: 'https://github.com/h2oai/wave', label: 'GitHub', position: 'right', },
+        { href: 'https://github.com/h2oai/wave',
+          position: 'right',
+          className: 'navbar-icon',
+          'aria-label': 'GitHub repository',
+          html: `<img src="https://img.icons8.com/ios-glyphs/30/000000/github.png" alt='Github Logo' />`,},
```

Comment:
Although this solves the problem, what I meant by absolute URLs was to be absolute within our docs so all there is needed to be done was to prefix your image src with `/`. See [third example here](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL#examples_of_absolute_urls). Please revert and only prefix the urls.

Having the SVG icons served from our own server is better performance-wise since we do not need to make another TLS handshake.